docs/2024/puzzles/day08.md

```diff
@@ -6,6 +6,129 @@ import Solver from "../../../../../website/src/components/Solver.js"
 
 https://adventofcode.com/2024/day/8
 
```

Comment:
please add an author credit for yourself, as seen previous days

Reply:
apologies, I am apparently discombobulated